src/topology/bounded_continuous_function.lean

```diff
@@ -665,6 +750,13 @@ norm_of_normed_group_le _ (mul_nonneg (norm_nonneg _) (norm_nonneg _)) _
 show that the space of bounded continuous functions from `α` to `β` is naturally a normed
 module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
+@[simps]
+def linear_isometry_continuous_map_of_compact [compact_space α] :
```

Comment:
Can you register in simp lemmas the different coercions of this one?

Otherwise, LGTM.

Reply:
Is https://github.com/leanprover-community/mathlib/pull/6701/commits/cb850f3b927506c83c632cdd1b34bbb068d093f3 what you meant?